conu/backend/docker/container.py

```diff
@@ -81,7 +81,8 @@ def __init__(self, container, mount_point=None):
     def __enter__(self):
         cmd = ["atomic", "mount", self.container.get_id(), self.mount_point]
         logger.debug(cmd)
-        run_cmd(cmd)
+        output = run_cmd(cmd, return_output=True)
```

Comment:
First instruction in `run_cmd()` is `logger.debug('command: "%s"' % ' '.join(cmd))` so the above `logger.debug(cmd)` can be also removed.

Reply:
Done, thanks.